source/_components/sensor.dht.markdown

```diff
@@ -15,6 +15,11 @@ ha_iot_class: "Local Polling"
 
 
 The `dht` sensor platform allows you to get the current temperature and humidity from a DHT11, DHT22, or AM2302 device.
+The user who starts Home Assistant needs the rights to access GPIOs. This can be done by adding it the GPIO group.
+
+```
+sudo adduser homeassistant gpio
```

Comment:
I would avoid this. Adding a hint/note is fine, but not everybody is using a setup where this applies (e.g., Hassio or Docker installs).